src/App.js

```diff
@@ -1,20 +1,20 @@
-import React from "react";
-import {Routes, Route, BrowserRouter} from 'react-router-dom';
-import Form from "./components/Form";
-import Navbar from "./components/Navbar";
-import Books from "./components/Books";
-import Categories from "./components/Categories";
+import React from 'react';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
+import Form from './components/Form';
+import Navbar from './components/Navbar';
+import Books from './components/Books';
+import Categories from './components/Categories';
 
 function App() {
   return (
     <BrowserRouter>
-        <Navbar />
-        <Routes>
-          <Route exact path="/" element={<Books />} />
-          <Route exact path="/form" element={<Form />} />
-          <Route exact path="/categories" element={<Categories />} />
-        </Routes>
-      </BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route exact path="/" element={<Books />} />
```

Comment:
- [ ] Great job with the routes, but as per the project requirements, there should be a route for BookList and this component should be the default page an not Books as you have implemented.
![image](https://user-images.githubusercontent.com/62849872/213615639-c2607e90-dd33-4a42-b0f1-3573af5ff2dd.png)

Reply:
Hi, @t-yanick I just wanna check because it feels like we have different instructions for the projects, 

![image](https://user-images.githubusercontent.com/114549019/213625269-8fabf0b6-5d3e-4d2b-a2b6-84d07c6345c5.png)

Because mine says to route two components; Books and categories, and then the booklist is a component inside the books component